go.mod

```diff
@@ -5,6 +5,8 @@ go 1.19
 // on release remove this wrangler replace and use the latest tag
 replace github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230705223603-201b4da5bdaf
 
+replace github.com/rancher/rancher/pkg/apis => github.com/Mbolotsuse/rancher/pkg/apis v0.0.0-20230804174703-16b9f3fe5bec
```

Comment:
This PR LGTM, whenever this fork is replacement, can be removed.

Reply:
Is this replacement being solved in https://github.com/rancher/rancher/pull/42319?

Reply:
Yep, once that merges I can make this point to a real rancher commit (Rather than my fork like it's doing now). But I need that one merged first.

Reply:
Thanks for the explanation. Anyway, LGTM. 👍🏻

Reply:
I've made the replacement, this now refers to a real rancher version.